pkg/actors/actors.go

```diff
@@ -216,11 +246,13 @@ func (a *actorsRuntime) Init() error {
 	}
 	appHealthFn := func() bool { return a.appHealthy.Load() }
 
-	a.placement = internal.NewActorPlacement(
-		a.config.PlacementAddresses, a.certChain,
-		a.config.AppID, hostname, a.config.HostedActorTypes,
-		appHealthFn,
-		afterTableUpdateFn)
+	if a.placement == nil {
+		a.placement = internal.NewActorPlacement(
+			a.config.PlacementAddresses, a.certChain,
+			a.config.AppID, hostname, a.config.HostedActorTypes,
+			appHealthFn,
+			afterTableUpdateFn)
+	}
```

Comment:
how about move this check to the `NewActors` function? 

Checking for nil gives the impression that placement can be nil in other places as well, that makes placement unsafe to call without a proper nil check. Overall I think that by unifying nil checks on the constructor would help the reader to understand which fields can be nil or not.

Reply:
fair enough